### PR TITLE
Revisit run tests charles

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

File: Lib/TestRunnerBase.py [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+        """
+           test_suite_name: test suite name
+           options        : options to use (in addition of default options always here)
+           options_files  : json files defining cdp/addparse argument definitions
+           get_sample_data: specifies whether sample data should be downloaded
+                            for the test run.
+           test_data_files_info: file name of a text file containing list of 
+                            data files needed for the test suite.
+        """
+        options_files.insert(0,os.path.join(sys.prefix,"share","cdat","cdat_runtests.json"))
```
Comment (muryanto1) [inline Lib/TestRunnerBase.py:47]: I am seeing <cdat_info repo dir>/share/cdat_runtests.json file, so should this line be:
options_files.insert(0,os.path.join(sys.prefix,"share","cdat_runtests.json"))
instead?

File: Lib/Const.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+#
+# The following constants are bits for storing run_tests.py argument options.
+# If user can run the testsuite and specify -H (for generate html), then 
+# the valid_options passed to TestRunnerBase constructor should have
+# OPT_GENERATE_HTML bit set.
+#
+# for boolean option arguments
+#
+OPT_GENERATE_HTML  = 0x00000001
```
Comment (muryanto1) [inline Lib/Const.py:13]: We can review all the OPT_* const as we will not use them.

Comment (doutriaux1) [reply Lib/Const.py:13]: yes I forgot to remove that file from the repo, I don't think it's needed any longer

File: Lib/TestRunnerBase.py (hunk 1 above)
Comment (doutriaux1) [reply Lib/TestRunnerBase.py:47]: nope if you look at setup.py I move it to share/cdat directory

File: Lib/Const.py (hunk 2 above)
Comment (muryanto1) [reply Lib/Const.py:13]: I use SUCCESS and FAILURE in TestRunnerBase.py and Util.py.

Comment (doutriaux1) [reply Lib/Const.py:13]: I moved them there, did I not push these changes?

File: Lib/Util.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+import sys
+import subprocess
+import shlex
+import Const
+import hashlib
+import time
+import multiprocessing
+import requests
+
+SUCCESS = 1
```
Comment (muryanto1) [inline Lib/Util.py]: This should be:
SUCCESS = 0

**Pull request merged**